### ISSUE TITLE
array(undefined, [42]) throws TypeError

### ISSUE THREAD

**Issue by mbostock** [d3/d3-interpolate#11]
As with #10, the starting value should be treated as an empty array if it’s not an array, rather than potentially throwing an error:

```
> d3_interpolate.value(undefined, [42])(0.5)
TypeError: Cannot read property 'length' of undefined
```

**Comment by krukon** [comment]
This pull request https://github.com/d3/d3-interpolate/pull/12 solves the problem with `undefined`/`null` case

**Comment by mbostock** [comment]
Fixed by #12.
